src/ansi-c/c_typecheck_initializer.cpp

```diff
@@ -350,8 +350,6 @@ void c_typecheck_baset::designator_enter(
   designator.push_entry(entry);
 }
 
-/// \param pre:initialized result, designator
-/// \return sets result
```

Comment:
Why is this being removed with no replacement?

Reply:
As explained in the commit message, 
> Obviously, it would be
better to add some up-to-date documentation, but until someone has time
to do that, this removes the unhelpful comment and stops Doxygen
generating 3 warnings.